src/frontend/komponenter/Behandling/Aktivitet/Aktivitet/AktivitetVisning.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { FC } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { IAktivitet } from '../../../../typer/overgangsstønad';
+import { VilkårStatus, VilkårStatusIkon } from '../../../Felleskomponenter/Visning/VilkårOppfylt';
+import { GridTabell } from '../../../Felleskomponenter/Visning/StyledTabell';
+import { Søknadsgrunnlag } from '../../../Felleskomponenter/Visning/DataGrunnlagIkoner';
+import { Element, Normaltekst, Undertittel } from 'nav-frontend-typografi';
+import { ArbeidssituasjonTilTekst, EArbeidssituasjon } from './typer';
+import SelvstendigNæringsdrivendeEllerFrilanser from './SelvstendigNæringsdrivendeEllerFrilanser';
+import Arbeidssøker from './Arbeidssøker';
+import { TidligereUtdanninger, UnderUtdanning } from './Utdanning';
+import { SeksjonWrapper } from '../../../Felleskomponenter/SeksjonWrapper';
+import Annet from './Annet';
+import Aksjeselskap from './Aksjeselskap';
+import { formaterNullableIsoDato } from '../../../../utils/formatter';
+import { ArbeidstakerLønnsmottakerSomFrilanser } from './ArbeidstakerLønnsmottakerSomFrilanser';
+
+interface Props {
+    aktivitet: IAktivitet;
+    vilkårStatus: VilkårStatus;
+}
+const AktivitetVisning: FC<Props> = ({ aktivitet, vilkårStatus }) => {
+    const {
+        arbeidssituasjon,
+        arbeidsforhold,
+        selvstendig,
+        aksjeselskap,
+        virksomhet,
+        arbeidssøker,
+        datoOppstartJobb,
+        underUtdanning,
+        tidligereUtdanninger,
+        særligeTilsynsbehov,
+        gjelderDeg,
+    } = aktivitet;
+
+    return (
+        <>
+            <SeksjonWrapper>
+                <GridTabell kolonner={3}>
+                    <VilkårStatusIkon className={'vilkårStatusIkon'} vilkårStatus={vilkårStatus} />
+                    <Undertittel className="tittel fjernSpacing">Aktivitet</Undertittel>
+                </GridTabell>
+                <GridTabell kolonner={3}>
+                    {arbeidssituasjon.includes(EArbeidssituasjon.erHjemmeMedBarnUnderEttÅr) && (
+                        <>
+                            <Søknadsgrunnlag />
+                            <Element className={'undertittel'}>
+                                {
+                                    ArbeidssituasjonTilTekst[
+                                        EArbeidssituasjon.erHjemmeMedBarnUnderEttÅr
+                                    ]
+                                }
+                            </Element>
+                        </>
+                    )}
+                </GridTabell>
+
+                <GridTabell kolonner={3}>
+                    {arbeidsforhold &&
+                        arbeidsforhold.map((arbeidsgiver) => (
+                            <ArbeidstakerLønnsmottakerSomFrilanser
+                                key={uuidv4()}
```

Comment:
alle `key={uuidv4()}` bør unngås - key bør være entydig for en rad hver gang den rendres.

Reply:
Fikset i ny commit